frontend/src/pages/Player/PlayerHook/PlayerHook.tsx

```diff
@@ -346,7 +346,7 @@ export const usePlayer = (): ReplayerContextInterface => {
 			const lastChunkIdx =
 				eventChunksData?.event_chunks[
 					eventChunksData?.event_chunks.length - 1
-				].chunk_index
+				]?.chunk_index
```

Comment:
Kind of surprised typescript didn't complain here.

Reply:
yeah, don't think typescript handles array out of bounds, though i think this only is out of bounds if `eventChunksData?.event_chunks.length` is 0